contracts/abi/LineaRollupV6.0.abi

```diff
@@ -557,40 +557,40 @@
       {
         "indexed": true,
         "internalType": "uint256",
-        "name": "lastBlockFinalized",
+        "name": "startBlockNumber",
+        "type": "uint256"
+      },
+      {
+        "indexed": true,
+        "internalType": "uint256",
+        "name": "endBlockNumber",
         "type": "uint256"
```

Comment:
I would like to consider adding `parentStateRootHash` and `finalStateRootHash` to this event to help with decentralization in the future. The operator may check by blob if It yields the expected `finalStateRootHash`. This would beneficial IMO and avoid further API changes and audits. 

@FlorianHuc @thedarkjester @julien-marchand WDYT?

Reply:
@jpnovais as discussed, the `parent is not accessible,` however the `finalStateRootHash` is. This has been added. Parent can be inferred by reading other events.